code/modules/client/preferences.dm

```diff
@@ -1489,7 +1489,7 @@ var/global/list/special_role_times = list( //minimum age (in days) for accounts
 								if(second_limb)
 									organ_data[second_limb] = "amputated"
 									rlimb_data[second_limb] = null
-						if("Prothesis")
+						if("Prosthesis")

```

Comment:
Looked at this very line dozens of times over hours since I redid the PR to resolve the first set of conflicts and it took a RANDOM GLANCE out of paranoia to find it x_X. Shiiiet.